src/main/java/org/opentripplanner/middleware/bugsnag/BugsnagDispatcher.java

```diff
@@ -103,44 +102,57 @@ private static String buildEventRequestFilter() {
      * Shorthand to create a Bugsnag event data request. This triggers an asynchronous job to prepare the data for one
      * single download. The returned event data request can be used to check the status of the request.
      *
-     * The “create an event data request” allows event data for a given organization to be collated in an asynchronous job
+     * The “create an event data request” allows event data for a given project to be collated in an asynchronous job
      * by Bugsnag. Once this job has completed a bespoke URL is provided where this data can be downloaded. Information on
      * this approach along with the filter parameters can be reviewed here:
      *
-     * https://bugsnagapiv2.docs.apiary.io/#reference/organizations/event-data-requests/create-an-event-data-request
+     * https://bugsnagapiv2.docs.apiary.io/#reference/projects/event-data-requests/create-an-event-data-request
      */
-    public static BugsnagEventRequest newEventDataRequest() {
-        return makeEventDataRequest(null);
+    public static BugsnagEventRequest newEventDataRequest(String projectId, int daysInPast) {
+        return makeEventDataRequest(null, projectId, daysInPast);
+    }
+
+    /**
+     * Shorthand to check a previously created Bugsnag event data request.
+     */
+    public static BugsnagEventRequest checkEventDataRequest(String eventDataRequestId, String projectId) {
+        return makeEventDataRequest(eventDataRequestId, projectId, -1);
     }
 
     /**
      * Get a previously created event data request (if id is non-null) OR create a new request. A status of 'complete'
      * signals that the requested data is ready to be downloaded from the populated url parameter.
      *
-     * More here: https://bugsnagapiv2.docs.apiary.io/#reference/organizations/event-data-requests/check-the-status-of-an-event-data-request
+     * More here: https://bugsnagapiv2.docs.apiary.io/#reference/projects/event-data-requests/check-the-status-of-an-event-data-request
      */
-    public static BugsnagEventRequest makeEventDataRequest(String eventDataRequestId) {
+    public static BugsnagEventRequest makeEventDataRequest(
+        String eventDataRequestId,
+        String projectId,
+        int daysInPast
+    ) {
         // Create new request if null ID is provided.
         boolean create = eventDataRequestId == null;
         URI eventDataRequestUri = HttpUtils.buildUri(
             BUGSNAG_API_URL,
-            "organizations", BUGSNAG_ORGANIZATION, "event_data_requests", eventDataRequestId
+            "projects", projectId, "event_data_requests", eventDataRequestId
         );
         LOG.debug("Making Bugsnag request: {}", eventDataRequestUri);
-
+        String filter = create
+            ? REQUEST_FILTERS.computeIfAbsent(daysInPast, BugsnagDispatcher::buildEventRequestFilter)
```

Comment:
I don't think saving these request filters like this is needed since the computation and memory savings are minimal.